genie-web/src/main/java/com/netflix/genie/web/resources/handlers/GenieResourceHttpRequestHandler.java

```diff
@@ -57,6 +60,12 @@
      */
     public static final String GENIE_JOB_ID_ATTRIBUTE = GenieResourceHttpRequestHandler.class.getName() + ".jobId";
 
+    /**
+     * The id of a job to use as a placeholder for all requests for V4 job output temporarily while we develop V4 job
+     * support.
+     */
+    public static final String V4_MOCK_JOB_ID = UUID.randomUUID().toString();
```

Comment:
This is a static constant that chances for every instance or every time the application starts.
Would it be less confusing to use an actual string, `mock-agent-job-id-logs-placeholder` or something?